src/components/AccountSaas/Create.tsx

```diff
@@ -94,165 +100,126 @@ export const AccountCreate = ({ children, ...props }: FlexProps) => {
+          </Trans>
+        </Text>
+        <Box px={{ base: 5, md: 10 }}>
+          <Box mb='32px'>
+            <MembershipSizeTypesSelector formValue='type' required={true} />
+          </Box>
+
+          <Box mb='32px'>
+            <OrganzationTypesSelector formValue='type' required={true} />
+          </Box>
+
+          <Box mb='32px'>
+            <CountriesTypesSelector formValue='type' required={true} />
+          </Box>
+        </Box>
+        <FormControl display='flex' alignItems='start' mb='12px'>
+          <Checkbox {...register('communications')} colorScheme='brandScheme' me='10px' mt='4px' />
+          <FormLabel mb='0' fontWeight='normal' color={textColor} fontSize='sm'>
+            <Trans i18nKey='create_org.communication'>
+              I want to receive communications and be contacted to tailor my governance experience.
+            </Trans>
           </FormLabel>
-        </Flex>
-        <FormErrorMessage>{errors.terms?.message?.toString()}</FormErrorMessage>
-      </FormControl>
-      <Button form='process-create-form' type='submit' isLoading={create} mx='auto' mb='32px' w='80%'>
-        {t('organization.create_org')}
-      </Button>
-      <Text color={textColorSecondary} fontSize='sm'>
-        <Trans i18nKey='create_org.already_profile'>
-          If your organization already have a profile, ask the admin to invite you to your organization.
-        </Trans>
-      </Text>
+        </FormControl>
+        <FormControl isInvalid={!!errors.terms} mb='32px'>
+          <Flex alignItems='start'>
+            <Checkbox {...register('terms', { required })} colorScheme='brandScheme' me='10px' mt='4px' />
+            <FormLabel mb='0' fontWeight='normal' color={textColor} fontSize='sm'>
+              <Trans
+                i18nKey='signup_agree_terms'
+                components={{
+                  termsLink: <Link as={ReactRouterLink} to='/terms' />,
+                  privacyLink: <Link as={ReactRouterLink} to='/privacy' />,
+                }}
+              />
+            </FormLabel>
+          </Flex>
+          <FormErrorMessage>{errors.terms?.message?.toString()}</FormErrorMessage>
+        </FormControl>
+        <Button form='process-create-form' type='submit' isLoading={create} mx='auto' mb='32px' w='80%'>
+          {t('organization.create_org')}
+        </Button>
+        <Text color={textColorSecondary} fontSize='sm'>
+          <Trans i18nKey='create_org.already_profile'>
+            If your organization already have a profile, ask the admin to invite you to your organization.
+          </Trans>
+        </Text>
 
-      {sent && error && (
-        <Alert status='error'>
-          <AlertIcon />
-          {error}
-        </Alert>
-      )}
-    </Flex>
+        {sent && error && (
+          <Alert status='error'>
+            <AlertIcon />
+            {error}
+          </Alert>
+        )}
+      </Flex>
+    </FormProvider>
   )
 }
 
```

Comment:
Just to know, on this file there are this component called `BasicAccountCreation` that is not imported anywhere. Is this old code? Can be removed?